planner_cspace_msgs/action/MoveFlex.action

```diff
@@ -0,0 +1,7 @@
+geometry_msgs/PoseStamped target_pose
+float64 goal_tolerance_lin
+float64 goal_tolerance_ang
+float64 goal_tolerance_ang_finish
```

Comment:
Could you add a comment to explain difference between `goal_tolerance_ang` and `goal_tolerance_ang_finish`?